pennylane/gradients/vjp.py

```diff
@@ -37,7 +37,7 @@ def _convert(jac, dy_row):
     return jac
 
 
-def compute_vjp_single_new(dy, jac):
+def compute_vjp_single_new(dy, jac, num=None):

```

Comment:
This `num` argument was added because TensorFlow does not know the shape of `dy` during the tracing process. Something similar was done for the old `compute_vjp` function.